table/tables/partition.go

```diff
@@ -242,3 +242,7 @@ func (t *partitionedTable) UpdateRecord(ctx sessionctx.Context, h int64, currDat
 	tbl := t.GetPartition(to)
 	return tbl.UpdateRecord(ctx, h, currData, newData, touched)
 }
+
+func (t *partitionedTable) GetID() int64 {
```

Comment:
irrelevant change?

Reply:
This change could prevent the potential wrong usage during refactor.